### PR TITLE
🌐 Add French translation for Features

### PR DESCRIPTION
**Pull request by Jefidev**
Hi everyone,

This PR contains the french translation for `docs/features.md` (issue #1972).

Here is a link to the [formatted document](https://github.com/Jefidev/fastapi/blob/master/docs/fr/docs/features.md).

Thanks :raccoon:

### CODE REVIEW

**Review by clemsau — APPROVED** [review]
LGTM 👌

File: docs/fr/docs/features.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+### Basé sur des standards ouverts
+
+* <a href="https://github.com/OAI/OpenAPI-Specification" class="external-link" target="_blank"><strong>OpenAPI</strong></a> pour la création d'API, incluant la déclaration de <abbr title="en français: routes. Aussi connu sous le nom anglais endpoints ou routes">path</abbr> <abbr title="Aussi connu sous le nom de méthodes HTTP. À savoir POST, GET, PUT, DELETE">operations</abbr>, paramètres, corps de requêtes, sécurité, etc.
+* Documentation automatique des modèles de données avec <a href="http://json-schema.org/" class="external-link" target="_blank"><strong>JSON Schema</strong></a> (comme OpenAPI est aussi basée sur JSON Schema).
+* Conçue avec ces standards après une analyse méticuleuse. Plutôt qu'en rajoutant des surcouches après coup.
+* Cela permet d'utiliser de la **génération automatique de code client** dans beaucoup de langages.
+
+### Documentation automatique
+
+Documentation d'API interactive et interface web d'exploration. Comme le framework est basé sur OpenAPI, de nombreuses options sont disponibles. Deux d'entre-elles sont inclus par défaut.
```
Comment (rjNemo) [inline docs/fr/docs/features.md:16]:
```suggestion
Documentation d'API interactive et interface web d'exploration. Comme le framework est basé sur OpenAPI, de nombreuses options sont disponibles. Deux d'entre-elles sont incluses par défaut.
```

File: docs/fr/docs/features.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+```
+
+!!! info
+    `**second_user_data` signifie:
+
+    Utilise les clés et valeurs du dictionnaire `second_user_data` directement comme des arguments clé-valeur. C'est équivalent à: `User(id=4, name="Mary", joined="2018-11-30")`
+
+### Support d'éditeurs
+
+Tout le framework a été conçus pour être facile et intuitif d'utilisation, toutes les décisions de design ont été testées sur de nombreux éditeurs avant même de commencer le développement final afin d'assurer la meilleure expérience de développement possible.
```
Comment (rjNemo) [inline docs/fr/docs/features.md:73]:
```suggestion
Tout le framework a été conçu pour être facile et intuitif d'utilisation, toutes les décisions de design ont été testées sur de nombreux éditeurs avant même de commencer le développement final afin d'assurer la meilleure expérience de développement possible.
```

File: docs/fr/docs/features.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+Dans le dernier sondages effectués auprès de développeurs python il était claire que <a href="https://www.jetbrains.com/research/python-developers-survey-2017/#tools-and-features" class="external-link" target="_blank">la fonctionnalité la plus utilisée est "l'autocomplètion"</a>.
```
Comment (rjNemo) [inline docs/fr/docs/features.md:75]:
```suggestion
Dans le dernier sondage effectué auprès de développeurs python il était clair que <a href="https://www.jetbrains.com/research/python-developers-survey-2017/#tools-and-features" class="external-link" target="_blank">la fonctionnalité la plus utilisée est "l'autocomplètion"</a>.
```

File: docs/fr/docs/features.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+
+Tout le framwork **FastAPI** a été conçus avec cela en tête. L'autocomplétion fonctionne partout.
```
Comment (rjNemo) [inline docs/fr/docs/features.md:77]:
```suggestion
Tout le framwork **FastAPI** a été conçu avec cela en tête. L'autocomplétion fonctionne partout.
```

File: docs/fr/docs/features.md [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+
+* dans <a href="https://code.visualstudio.com/" class="external-link" target="_blank">Visual Studio Code</a>:
+
+![editor support](https://fastapi.tiangolo.com/img/vscode-completion.png)
+
+* dans <a href="https://www.jetbrains.com/pycharm/" class="external-link" target="_blank">PyCharm</a>:
+
+![editor support](https://fastapi.tiangolo.com/img/pycharm-completion.png)
+
+Vous aurez des propositions de complétion que vous n'auriez jamais imaginé. Par exemple la clé `prix` dans le corps d'un document JSON (qui est peut-être imbriqué) venant d'une requête.
```
Comment (rjNemo) [inline docs/fr/docs/features.md:91]:
```suggestion
Vous aurez des propositions de complétion que vous n'auriez jamais imaginées. Par exemple la clé `prix` dans le corps d'un document JSON (qui est peut-être imbriqué) venant d'une requête.
```

File: docs/fr/docs/features.md [hunk 6]
```diff
@@ -0,0 +1,201 @@
...
+* HTTP Basic.
+* **OAuth2** (aussi avec **JWT tokens**). Jetez un oeil au tutoriel [OAuth2 avec JWT](tutorial/security/oauth2-jwt.md){.internal-link target=_blank}.
+* Clés d'API dans:
+    * Le header.
+    * Les paramètres de requêtes.
+    * Les cookies, etc.
+
+Plus toutes les fonctionnalités de sécurités venant de Starlette (incluant les **cookies de sessions**).
+
+Le tout conçu en composant réutilisable facilement intégrable a vos systèmes, data stores, base de données relationnelle ou NoSQL, etc.
```
Comment (rjNemo) [inline docs/fr/docs/features.md:132]:
```suggestion
Le tout conçu en composant réutilisable facilement intégrable à vos systèmes, data stores, base de données relationnelle ou NoSQL, etc.
```

File: docs/fr/docs/features.md [hunk 7]
```diff
@@ -0,0 +1,201 @@
...
+* 100% de couverture de test.
+* 100% d'annotations de types dans le code.
+
+## Fonctionnalités de Pydantic
+
+**FastAPI** est totalement compatible avec (et basé sur) <a href="https://pydantic-docs.helpmanual.io" class="external-link" target="_blank"><strong>Pydantic</strong></a>. Le code utilisant Pydantic que vous ajouterez fonctionnera donc aussi.
+
+Inclus des librairies externes basées, aussi, sur Pydantic, servent d'<abbr title="Object-Relational Mapper">ORM</abbr>s, <abbr title="Object-Document Mapper">ODM</abbr>s pour les bases de données.
+
+Cela signifie aussi que, dans la plupart des cas, vous pouvez fournir l'objet reçus d'une requête **directement à la base de données**, comme tout est validé automatiquement.
```
Comment (rjNemo) [inline docs/fr/docs/features.md:182]:
```suggestion
Cela signifie aussi que, dans la plupart des cas, vous pouvez fournir l'objet reçu d'une requête **directement à la base de données**, comme tout est validé automatiquement.
```

**Comment by rjNemo** [comment]
Very good translation.
Just a few comments.

**Comment by ClemSau** [comment]
Thanks for the rectifications

**Comment by Jefidev** [comment]
Thanks for the review. I committed your suggestions.

**Review by rjNemo — APPROVED** [review]

**Comment by tiangolo** [comment]
Awesome, thank you @Jefidev! :cake: :bow: 

And thanks for the reviews @rjNemo and @ClemSau ! :nerd_face:

**Pull request merged**